docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -160,10 +174,10 @@ Start Grafana by running:
 - The default configuration specifies a SQLite3 db at `/var/lib/grafana/grafana.db`
 - Installs HTML/JS/CSS and other Grafana files at `/usr/share/grafana`
 
-## Next steps
+## Upgrade Grafana
 
-Refer to the [Getting Started]({{< relref "../../../getting-started/build-first-dashboard/" >}}) guide for information about logging in, setting up data sources, and so on.
+While the process for upgrading Grafana is very similar to installing Grafana, there are important backup tasks you should perform. Refer to [Upgrade Grafana]({{< relref "../../../upgrade-guide/" >}}) for guidance on updating an existing installation.
 
-## Configure Grafana
+## Next steps
 
-Refer to the [Configuration]({{< relref "../../configure-grafana/" >}}) page for details on options for customizing your environment, logging, database, and so on.
+- [Start the Grafana server]({{< relref "../../../restart-grafana/" >}})
```

Comment:
next steps should be to configure grafana

Reply:
keeping as is based on our discussion.